src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -46,22 +48,22 @@ public void RunApplication(Application app)
                 Dynamo.Applications.StartupUtils.ASMPreloadFailure += ASMPreloadFailureHandler;
                 if (!String.IsNullOrEmpty(ASMPath))
                 {
-                    model = Dynamo.Applications.StartupUtils.MakeModel(false,ASMPath);
+                    model = Dynamo.Applications.StartupUtils.MakeModel(false, ASMPath, hostName);
                 }
                 else
                 {
-                    model = Dynamo.Applications.StartupUtils.MakeModel(false);
+                    model = Dynamo.Applications.StartupUtils.MakeModel(false, string.Empty, hostName);
                 }
```

Comment:
NIT: We could just use null coalesce, i.e. `ASMPath ?? string.empty`, to avoid the if statement.